webui/react/src/components/ExperimentActionDropdown.tsx

```diff
@@ -223,7 +245,9 @@ const ExperimentActionDropdown: React.FC<Props> = ({
             ExperimentRetainLogsModal.open();
             break;
           case Action.HyperparameterSearch:
-            HyperparameterSearchModal.open();
+            InterstitialModal.open();
+            onVisibleChange?.(false);
+            await fetchedExperimentItem();
```

Comment:
given that you're handling the fetch error above, you should be able to get away with just
```suggestion
            fetchedExperimentItem();
```

Reply:
fixed